### PR TITLE
Added Flurry AppMetrics source

### PR DESCRIPTION
**Pull request by jkehler**
Source that connects to Flurry AppMetrics API.

### CODE REVIEW

File: lib/optimus_prime/sources/flurry_appmetrics.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      end
+
+      private
+
+      def get_report
+        handle_response RestClient.get(@url, params: @params)
+      end
+
+      def handle_response(response)
+        return JSON.parse(response) unless response.code != 200
```
Comment (Anumodana) [inline lib/optimus_prime/sources/flurry_appmetrics.rb]: Why don't you just check `if response.code == 200` ?

Comment (jkehler) [inline lib/optimus_prime/sources/flurry_appmetrics.rb]: Not exactly sure what you mean. It returns the parsed JSON if status is 200 else it flows down inside the function to log the error and raise an exception.

Edit. Nevermind. i think i know what you mean.

File: spec/optimus_prime/sources/flurry_appmetrics_spec.rb [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+
+  context 'invalid version paramater' do
+    let(:metric_name) { 'ActiveUsers' }
+    let(:country) { nil }
+    let(:group_by) { nil }
+    let(:version) { 111 }
+
+    let(:response_body) { File.read 'spec/supports/flurry/appmetrics_invalid_version.json' }
+
+    it 'should return a valid json response' do
```
Comment (Anumodana) [inline spec/optimus_prime/sources/flurry_appmetrics_spec.rb]: Did you forget to change the description?

**Comment by Anumodana** [comment]
It looks good to me. :+1:

**Pull request merged**